multiversx_sdk_cli/cli_contracts.py

```diff
@@ -28,14 +30,16 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
 
     sub = cli_shared.add_command_subparser(subparsers, "contract", "new",
                                            "Create a new Smart Contract project based on a template.")
-    sub.add_argument("name")
+    sub.add_argument("--name", help="The name of the contract. If missing, the name of the template will be used.")
     sub.add_argument("--template", required=True, help="the template to use")
-    sub.add_argument("--directory", type=str, default=os.getcwd(),
-                     help="🗀 the parent directory of the project (default: current directory)")
+    sub.add_argument("--tag", help="the framework version on which the contract should be created")
```

Comment:
maybe specify the version that will be used in case that the flag is not set

Reply:
That version is somehow out of `mxpy`'s control.

Reply:
Indeed. I was trying to figure out if there is a simple way to do that, but I think it's to much work for what it adds.